lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word
+
+  def initialize(word)
+    @word = word
+  end
+
+  def score
+    @point_values(letter)
+        letters = letter.chars.map.upcase do |char|
+          point_values(letters)
```

Comment:
Remember to use square brackets to access a hash. 
For example 
point_values[lchar]

Reply:
Also make sure that the variables you use stay the same.  What you have now is kind of like this:
letters.map do |char|
   point_values[letter]
end 
Where what you want is more like this:
letters.map do |letter|
    point_values[letter]
end